rest-service/manager_rest/test/endpoints/test_deployment_groups.py

```diff
@@ -359,6 +359,36 @@ def test_remove_from_filters(self):
         group = self.client.deployment_groups.get('group1')
         assert group.deployment_ids == []
 
+    def test_set_labels(self):
+        """Shrink a group providing filter_id"""
```

Comment:
does this docstring make sense in this place?

Reply:
whoops